decimal.md

```diff
@@ -27,11 +27,9 @@ When inserting a decimal value:
 
 ## Format
 
-When inserting a decimal value, format it as `DECIMAL '1.2345'`. This casts the value as a string, which preserves the number's exact precision.
+When inserting into a `DECIMAL` column, format the value as `1.2345`. CockroachDB considers the type of the column when handling literals, so non-decimal numbers are also accepted.
```

Comment:
Oh, I read below and I think I understand what you meant here now. I still dont think I would say "considers the type of the column" though, because the type system is a lot more than just looking at the column (that's just a simple example). Something along the lines of "considers how the value is used" would be more descriptive.